src/components/sections/HeroStats.section.tsx

```diff
@@ -56,17 +56,18 @@ const HeroStatSection: React.FC = () => {
                     alt="Hawkhacks stats"
                     className="block w-full sm:hidden"
                 />
-                <div className="space-y-1 pb-4 pl-10 text-left sm:pl-24 sm:pt-12 lg:pl-16 lg:pt-16 2xl:pl-60 2xl:pt-24">
-                    <img
-                        className="inline-block aspect-square w-16 sm:mx-0 lg:w-28 xl:w-32 2xl:w-52"
-                        src={HawkHacksLogo}
-                        alt="Hawkhacks logo"
-                    />
-                    <div className="sm:space-y-2">
-                        <h1 className="bg-gradient-to-b from-[#2B6469] to-[#00CEDB] bg-clip-text  text-4xl font-extrabold text-transparent sm:text-5.5xl lg:text-7xl xl:text-8.5xl">
+                <div className="mx-auto mt-16 px-4 sm:ml-16 sm:mt-16 sm:px-0 lg:ml-20 lg:mt-24 xl:ml-24 xl:mt-32 2xl:mt-10">
+                    <div>
+                        {/* lg:w-36 */}
```

Comment:
Not sure if you meant to keep it, but remove this comment if its no longer needed.

Reply:
noted :)